lib/rucio/web/rest/flaskapi/v1/common.py

```diff
@@ -210,6 +210,15 @@ def parse_scope_name(scope_name: str, vo: Optional[str]) -> tuple[str, ...]:
     :raises ValueError: when scope_name could not be parsed.
     :returns: a (scope, name) tuple.
     """
+
+    encoded_slash = r"%2F"
+    if encoded_slash in scope_name:
+        if scope_name.count('/') != 1:
+            raise ValueError(f"Could not parse '{scope_name}' ({scope_name=}) with encoded '/' into scope and name.")
+        scope, name = scope_name.split('/', 1)
+        name = name.replace(encoded_slash, '/')
```

Comment:
Small edge case, but we probably want to prevent people from having `%2F` in their scope or name (which the current `SCOPE_NAME_REGEXP` allows), otherwise this would break.

Reply:
Good point, but actually the DID name regex will not allow `%2F` in the request as `%` is forbidden (atleast in the default settings). It is true that on a given policy package this could be an issue, but it's a more marginal possibility that can always be addressed by updating the did regex.